app/web/features/profile/locales/en.json

```
@@ -127,37 +128,68 @@
   "report_reason": "Reason",
   "report_user": "Report this user",
   "leave_reference": {
+    "add_a_reference": "Add a reference to {{name}}'s profile",
     "appropriate_behavior": "Appropriate Behavior",
-    "appropriate_explanation": "Awesome –– We hope you had a great time! To help keep our community safe, we want to ask about your interaction with your fellow Coucher.",
+    "appropriate_explanation": "Your private feedback on this page, and your public reference on the next page, are vital to keeping the community safe, trustworthy, and welcoming. Thanks for doing your part to help the community!",
     "appropriate_question": "Did you feel safe with this person's behavior?",
+    "by_writing_thoughtful": "Not sure what to write? Read our guide <1>here</1>. By writing honestly and in detail about your experience with this person, you will directly help other members and contribute to safety and trust in the Couchers community.",
     "coucher_was_appropriate": "Yes, this person's behavior was appropriate.",
     "coucher_was_not_appropriate": "No, this person's behavior was not appropriate.",
-    "contact_text": "If you have any questions or wish to provide additional information, please don't hesitate to <1>contact us here.</1>",
+    "did_stay_confirmation": {
+      "title_hosted": "Confirm {{name}} did not stay",
+      "title_surfed": "Confirm you did not stay with {{name}}",
+      "message_hosted": "You <strong>will not be able to write a reference</strong> if you indicate {{name}} didn't stay with you. They will not be notified but they can still leave a reference for you.<br /><br />If you need to report something find out how <2>here</2>.",
+      "message_surfed": "You <strong>will not be able to write a reference</strong> if you indicate you didn't stay with {{name}}. They will not be notified but they can still leave a reference for you.<br /><br />If you need to report something find out how <2>here</2>.",
+      "confirm_hosted": "Confirm I didn't host",
+      "confirm_surfed": "Confirm I didn't surf"
+    },
+    "did_stay_explanation": "Select 'No' if you didn't actually meet. You will not be able to write a public reference, but can optionally share <strong>private</strong> feedback with us about what happened. The other person won’t be informed, but can still leave a reference if they say you met.",
     "host_request_reference_explanation": "Host request references are only made visible once both references have been written, or after 2 weeks. Hold tight!",
```

"Hold tight!" is a bit awkward here/might not be understood by non-native speakers.

```suggestion
    "host_request_reference_explanation": "Host request references are only made visible once both references have been written, or after 2 weeks.",
```